Header header
float64 fx
float64 fz
float64 tauSpringA
float64 tauSpringB
float64 dtauSpringA
float64 dtauSpringB
